src/networks.json

```diff
@@ -1795,5 +1795,21 @@
     },
     "start": 1126886,
     "logo": "ipfs://QmaARJiAQUn4Z6wG8GLEry3kTeBB3k6RfHzSZU9SPhBgcG"
+  },
+  "900000": {
+    "key": "900000",
+    "name": "Posichain Mainnet",
+    "shortName": "PSC",
+    "chainId": 900000,
+    "network": "mainnet",
+    "multicall": "0xbe779BaA9dC15Ef307bdf4dDb35440A0a421a89A",
```

Comment:
Is there a way to check the code here https://ubiqscan.io/address/0x6668750957e4083725926b71c41bdf1434c73262

Everything else works good

Reply:
Why we need to check that code? about multicall contract, you can check here: https://explorer.posichain.org/address/0xbe779BaA9dC15Ef307bdf4dDb35440A0a421a89A?activeTab=7

Reply:
@ChaituVR please help to check

Reply:
Oops wrong link paste, Thanks @dannyposi